terraform/modules/ecr/README

```diff
@@ -0,0 +1,28 @@
+DevOps Academy Project 3 Jun 2020 ECR Repo
+
+Instructions:
+
+1. After apply terraform get the repository URL
+
+terraform output | grep repository_url
```

Comment:
Great job 🚀 just one comment. Is there a possibility for us to use the outputs.tf to return the repository URL and use it in the other template as `module.<MODULE NAME>.<OUTPUT NAME>`
Have a look here to see if its possible. [https://www.terraform.io/docs/configuration/outputs.html](url)

Reply:
I've written the output accordingly to this documentation. I suggested use grep to filter the repository URL on terraform output command.

Reply:
@jayanath I improved it using jq to filter the repository variable value. I will use it on docker Makefile